src/external-site/_vars-external-site.html

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
     // Find the external-urls in your local cfgov-refesh repo.
-    find . -name '*.html' ! -path './src/*' ! -path './node_modules/*' ! -path './_tests/*' \
+    find . -name '*.html' ! -path './*' ! -path '../node_modules/*' ! -path '../test/*' \
```

Comment:
Updated paths in https://github.com/cfpb/cfgov-refresh/commit/cca1733c3ae120ee9ebb47d8780932f2d8f67b11